packages/hint/docs/contributor-guide/guides/create-custom-hint.md

```diff
@@ -32,7 +32,8 @@ the CLI.
 >* Make executable release. run `yarn webpack`.
 >* Now run `npm create hint` from `packages` folder.
 >
->The generated files will be slighthly different (references and task will change a bit).
+>The generated files will be
+>slightly different (references and task will change a bit).
```

Comment:
If you can, you may want to split this markdown fix into its own commit. 

Context: When we're ready to merge this I'd like to squash down to two commits - one for everything under `packages/create-hint` with the `New:` label and another for the doc updates under `packages/hint` with the `Docs:` label. This will help our release script bump versions appropriately.

Reply:
Ahh yes right. Let me do this.

Reply:
@antross I have re-based this PR to 2 commits. One is for docs and another one for updates.

Reply:
Thanks! One minor tweak, let's use `New:` instead of `Chore:` for the commit with the functionality updates. We're adding support for localization of official hints to `create-hint` so I'd consider it a new feature.